src/gsi/gsi_rfv3io_mod.f90

```diff
@@ -2382,6 +2508,7 @@ subroutine gsi_fv3ncdf_write_v1(filename,varname,var,mype_io,add_saved)
     real(r_kind),allocatable,dimension(:,:,:):: work_sub,work_a
     real(r_kind),allocatable,dimension(:,:,:):: work_b
     real(r_kind),allocatable,dimension(:,:):: workb2,worka2
+    integer ii
```

Comment:
Please add (i_kind) to integer declaration.

Reply:
the second & has been removed

Reply:
deleted integer ii'